File: API/Entities/Order.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace API.Entities
{
    public class Order
    {
        public int Id { get; set; }
        public Unit? Unit { get; set; }
        public AppUser? Client { get; set; }
        public AppUser? DeliveryMan { get; set; }
        public Location? DeliveryLocation { get; set; }
        public DateTime DeliveryDate { get; set; }
        public DateTime ReturnDate { get; set; }
        public bool DeliveryInProcess { get; set; } 
        public bool Cancelled { get; set; }
        public bool DeliveryCompleted { get; set; }
        public bool ClientGotDelivery { get; set; }
        
        public bool InUsage { get; set; }

        public AppUser? ReturnDeliveryman { get; set; }
        public bool UnitReturned { get; set; }
        public Location? ReturnFromLocation { get; set; }
        public Point? ReturnPoint { get; set; }
    }
}

File: CLIENT/src/app/models/order.ts
import { Client } from "./client"
import { Point, Unit } from "./item"
import { UserLocation } from "./user"

export interface Order{
    id: number
    unit: Unit
    client: Client
    deliveryMan: Client
    deliveryLocation: UserLocation
    deliveryDate: string
    returnDate: string
    deliveryInProcess: boolean
    cancelled: boolean
    deliveryCompleted: boolean
    clientGotDelivery: boolean
    inUsage: boolean
    returnDeliveryman: Client
    unitReturned: boolean
    returnFromLocation: UserLocation
    returnPoint: Point
}